syncd/tests.cpp

```diff
@@ -264,18 +266,24 @@ void test_bulk_next_hop_group_member_create()
 
     for (uint32_t i = 0; i <  count; ++i)
     {
+        // sidlist object
+        sai_object_id_t sidlist;
+        sai_attribute_t sidattr[1] = { };
+        sidattr[0].id = SAI_SRV6_SIDLIST_ATTR_TYPE;
+        sidattr[0].value.s32 = SAI_SRV6_SIDLIST_TYPE_INSERT_RED;
+        status = sai_srv6_api->create_srv6_sidlist(&sidlist, switch_id, 1, sidattr);
+
         sai_object_id_t hop_vid;
 
-        sai_attribute_t nhattr[3] = { };
+        sai_attribute_t nhattr[2] = { };
 
         nhattr[0].id = SAI_NEXT_HOP_ATTR_TYPE;
-        nhattr[0].value.s32 = SAI_NEXT_HOP_TYPE_SEGMENTROUTE_ENDPOINT;
-
-        nhattr[1].id = SAI_NEXT_HOP_ATTR_SEGMENTROUTE_ENDPOINT_TYPE;
+        nhattr[0].value.s32 = SAI_NEXT_HOP_TYPE_SRV6_SIDLIST;
 
-        nhattr[2].id = SAI_NEXT_HOP_ATTR_SEGMENTROUTE_ENDPOINT_POP_TYPE;
+        nhattr[1].id = SAI_NEXT_HOP_ATTR_SRV6_SIDLIST_ID;
+        nhattr[1].value.oid = sidlist;
 
-        status = sai_next_hop_api->create_next_hop(&hop_vid, switch_id, 3, nhattr);
+        status = sai_next_hop_api->create_next_hop(&hop_vid, switch_id, 2, nhattr);
```

Comment:
create new test method for this, you put this in test_bulk_next_hop_group_member_create

Reply:
@kcudnik This is the existing test that uses segmentroute object which got renamed to srv6 object in the latest SAI.